pasqal_cloud/client.py

```diff
@@ -108,9 +107,8 @@ def _request(
             headers={"content-type": "application/json"},
             auth=self.authenticator,
         )
+        rsp.raise_for_status()
         data: JSendPayload = rsp.json()
-        if rsp.status_code >= 400:
-            raise HTTPError(data)
```

Comment:
From what I can tell, it looks like this error is now unused in our codebase. There is only one extra reference where we catch it but it is not raised so it looks we should get rid of it